README.md

```diff
@@ -237,121 +328,122 @@ export LUA_PATH="/usr/share/lua/5.1/?.lua;/usr/share/lua/5.1/?/init.lua;./?.lua;
 lua /usr/lib/sqm-autorate/sqm-autorate.lua
 ```
 
-(_Is this next paragraph correct?_) When you run a speed test, you should see the `current_dl_rate` and
-`current_ul_rate` values change to match the current conditions.
-They should then drift back to the configured download and update rates
-when the link is idle.
+To view the current CAKE speeds, use the bash command
+
+```bash
+tc qdisc
+```
 
 The script logs information to `/tmp/sqm-autorate.csv` and speed history data to `/tmp/sqm-speedhist.csv`.
 See the [Verbosity](#Verbosity_Options) options (below)
 for controlling the logging messages.
 
-#### Service Execution (for Steady-State Execution)
+## Upgrading
+`sqm-autorate` is frequently updated in response to test reports and user requests.
+The following `bash` commands will upgrade it to the latest version
+1. Stop the service
+   ```bash
+   service sqm-autorate stop
+   ```
+2. Run the setup script
+   ```bash
+   sh -c "$(wget -q -O- https://raw.githubusercontent.com/sqm-autorate/sqm-autorate/testing/lua-threads/sqm-autorate-setup.sh)"
+   ```
+3. Start the service
+   ```bash
+   service sqm-autorate start
+   ```
+
+## Removal
 
-As noted above, the setup script installs `sqm-autorate.lua` as a service,
-so that it starts up automatically when you reboot the router.
+_(We hope that you will never need to uninstall this autorate program, but if you want to...)_
+Run the following removal script to remove the operational files:
 
 ```bash
-service sqm-autorate enable && service sqm-autorate start
+sh -c "$(wget -q -O- https://raw.githubusercontent.com/sqm-autorate/sqm-autorate/testing/lua-threads/sqm-autorate-remove.sh)"
 ```
 
-### Output and Monitoring
+## Development
 
-#### View of Processes
+This _sqm-autorate_ program is written primarily for OpenWrt 21.02.
```

Comment:
@richb-hanover wanted to rewrite this, but I can't recall where he wrote his version